File: clients/typescript/src/SystemdManagerDisableUnitsRequest.ts

export class SystemdManagerDisableUnitsRequest {
  private _files: string[];

  constructor(input: {
    files: string[],
  }) {
    this._files = input.files;
  }

  get files(): string[] { return this._files; }
  set files(files: string[]) { this._files = files; }
}

File: clients/rust/src/systemd_manager_disable_units_request.rs
// SystemdManagerDisableUnitsRequest represents a SystemdManagerDisableUnitsRequest model.
#[derive(Clone, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
pub struct SystemdManagerDisableUnitsRequest {
    #[serde(rename="files")]
    pub files: Vec<String>,
}

impl SystemdManagerDisableUnitsRequest {
    pub fn new(files: Vec<String>) -> SystemdManagerDisableUnitsRequest {
        SystemdManagerDisableUnitsRequest {
            files,
        }
    }
}
